astrovascpy/bloodflow.py

```diff
@@ -478,15 +478,8 @@ def simulate_vasodilation_ou_process(graph, dt, nb_iteration, nb_iteration_noise
         endfeet_id = graph.edge_properties.loc[:, "endfeet_id"].to_numpy()
 
     # Distribute vectors across MPI ranks
-    radius_origin_petsc = array2PETScVec(radius_origin if MPI_RANK == 0 else [])
-    endfeet_id_petsc = BinaryIO2PETScVec(endfeet_id if MPI_RANK == 0 else [], "tempVec.dat")
-
-    # Each MPI rank/core deals with a chunk of the global vectors
-    radius_origin = radius_origin_petsc.getArray()
-    radius_origin = radius_origin.astype(FLOAT)
-
-    endfeet_id = endfeet_id_petsc.getArray()
-    endfeet_id = endfeet_id.astype(int)
+    radius_origin = distribute_array(radius_origin if MPI_RANK == 0 else [])
+    endfeet_id = distribute_array(endfeet_id if MPI_RANK == 0 else [])
```

Comment:
Isn't `None` enough?
```suggestion
    radius_origin = distribute_array(radius_origin if MPI_RANK == 0 else None)
    endfeet_id = distribute_array(endfeet_id if MPI_RANK == 0 else None)
```

Reply:
Yes. It is fine